scrape/scrape.py

```diff
@@ -66,6 +66,7 @@ def __init__(self, resource: str = "", region: str = "", account: str = "") -> N
     "AWS Elastic Load Balancing V2": "elasticloadbalancing",
     "AWS IoT Greengrass V2": "greengrass",
     "AWS Marketplace Catalog": "aws-marketplace",
+    "AWS Marketplace Deployment Service": "aws-marketplace",
```

Comment:
The automated Github action has been failing due to discovering multiple service names for `aws-marketplace`, and this fixes that issue. Feel free to just take this line!